app/views/my_facilities/blood_pressure_control.html.erb

```diff
@@ -1,209 +1,318 @@
+                  </th>
+                <% end %>
+              </tr>
+            </thead>
+            <tbody>
+              <% @facilities_by_size[size].each do |facility| %>
+                <% six_month_rate_change = @data_for_facility[facility.name]["controlled_patients_rate"][@period] - @data_for_facility[facility.name]["controlled_patients_rate"][@start_period] || 0 %>
+                <tr data-row="<%= facility.slug %>" data-trend-color="<%= six_month_rate_change >= 0 ? "green" : "red" %>">
+                  <td class="type-title">
+                    <%= link_to(reports_region_path(facility, report_scope: "facility")) do %>
+                      <%= facility.name %>
+                    <% end %>
+                  </td>
+                  <td>
+                    <%= number_with_delimiter(@data_for_facility[facility.name]["adjusted_registrations"].values.last || 0) %>
+                  </td>
+                  <td>
+                    <%= number_with_delimiter(@data_for_facility[facility.name]["cumulative_registrations"].values.last || 0) %>
+                  </td>
+                  <td>
+                    <div class="w-80px h-20px">
+                      <canvas id=<%= facility.slug %>></canvas>
+                    </div>
+                  </td>
+                  <td class="fw-bold ta-center <%= six_month_rate_change >= 0 ? "c-green-dark" : "c-red-dark" %>">
+                    <%= six_month_rate_change >= 0 ? "+" : "" %><%= number_to_percentage(six_month_rate_change, precision: 0) %>
+                  </td>
+                  <% (@start_period..@period).each do |period| %>
+                    <td>
+                      <%= @data_for_facility[facility.name]["controlled_patients"][period] %>
+                    </td>
+                    <td class="type-percent">
+                      <% controlled_patients_rate = @data_for_facility[facility.name]["controlled_patients_rate"][period] %>
+                      <em data-control-rate="<%= controlled_patients_rate %>">
+                        <%= number_to_percentage(controlled_patients_rate || 0, precision: 0) %>
+                      </em>
+                    </td>
+                  <% end %>
+                </tr>
               <% end %>
-              <input class="selected-quarter" name="registration_quarter" value="<%= registration_quarter %>" form="query-filters" hidden>
-              <input class="selected-month" name="registration_month" value="<%= registration_month %>" form="query-filters" hidden>
-              <input class="selected-year" name="registration_year" value="<%= registration_year %>" form="query-filters" hidden>
-              <input class="selected-cohort-period" name="cohort_period" value="<%= cohort_period %>" form="query-filters" hidden>
+            </tbody>
+          </table>
+        <% end %>
+      </div>
+    </div>
+  </div>
+  <script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
```

Comment:
everything above can go into a new partial, and everything in the else can go into another partial as well.

Basically, separate out the old view code from the new and name the partials after that...maybe `bp_control_details_v1` and `bp_control_details_v2`